src/main/resources/wayf_widget_template.js

```diff
@@ -62,18 +50,6 @@ function readLocalId() {
     return readCookieValue(LOCAL_ID_COOKIE_NAME);
 }
 
-function setLocalId(localIdValue) {
```

Comment:
The widget methods are not accessed by any client code. The client code generates the localID on the server side. It transfers it to the browser with a cookie inside the header.